cmake/compiler/arcmwdt/generic.cmake

```diff
@@ -19,6 +19,7 @@ if(NOT CMAKE_DTS_PREPROCESSOR)
 endif()
 
 find_program(CMAKE_C_COMPILER ${CROSS_COMPILE}ccac PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+find_program(CMAKE_GCOV ${CROSS_COMPILE}gcov PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
```

Comment:
ARC MWDT doesn't have gcov. It is LLVM based and have `llvm-cov` - so we can copy the code chunk from clang.